pontoon/base/vcs_models.py

```diff
@@ -0,0 +1,127 @@
+"""
+Models for working with remote translation data stored in a VCS.
+"""
+import os.path
+from itertools import chain
+
+
+class VCSProject(object):
```

Comment:
Not until now!

I mean, they are undoubtly modeling the data that we work with, and they are a nice counterpoint to the DB-based models. But if your point is that calling them models is potentially misleading, I see that. I'm not sure if there's a better name for them than models, though.